schema/me/sale_registrations.js

```diff
@@ -0,0 +1,45 @@
+import { first } from 'lodash';
+import gravity from '../../lib/loaders/gravity';
+import Sale from '../sale';
+import Sales from '../sales';
+import Bidder from '../bidder';
+import {
+  GraphQLList,
+  GraphQLBoolean,
+  GraphQLObjectType,
+} from 'graphql';
+
+export const SaleRegistrationType = new GraphQLObjectType({
+  name: 'SaleRegistration',
+  fields: () => ({
+    is_registered: {
+      type: GraphQLBoolean,
+    },
+    bidder: {
+      type: Bidder.type,
+    },
+    sale: {
+      type: Sale.type,
+    },
+  }),
+});
+
+export default {
+  type: new GraphQLList(SaleRegistrationType),
+  args: Sales.args,
+  resolve: (root, options, { rootValue: { accessToken } }) => {
+    return gravity('sales', options)
```

Comment:
Wouldn't you want to query `auction_state=open` here (dunno if you can do that in Gravity)? Otherwise it would just pull the latest 10 sales and :pray: they match up with the right bidders.

I think you want to go about this in the other direction `me/bidders` -> `bidders.map(bidder => gravity(`sale/${bidder.sale._id}`)`